src/lib/skip.ts

```diff
@@ -71,7 +71,8 @@ export const fetchSkipStatus = async ({ transactionHash, chainId, baseUrl }: Ski
 };
 
 const getTransferFromStatusResponse = (skipStatusResponse: TxStatusResponseJSON) => {
-  return skipStatusResponse.transfers[0];
+  // todo - are we sure this is always present?
```

Comment:
hm i wouldnt count on it tbh, could be in a followup but id throw an error if its not defined

Reply:
okie